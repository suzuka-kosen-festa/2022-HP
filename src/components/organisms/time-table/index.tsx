import type { FC } from "react";
import tw from "twin.macro";
import { useMemo } from "react";
import type {
  TimeTableProperties,
  TimetableHeadProperties,
  TimeTableColumnProperties,
  TimeTableAxisProperties,
} from "./types/model";
import { TimeTableRow } from "../../molecules/time-table-row";

const TimeTableHeadBox = tw.div`flex justify-center items-center px-0.5 py-2 sm:(px-1 py-4) md:(px-3 py-6) lg:(px-14 py-21.25)`;

const TimeTableHead: FC<TimetableHeadProperties> = ({ stage }) => {
  // eslint-disable-next-line consistent-return
  const memo = useMemo(() => {
    switch (stage) {
      case "main":
        return { style: tw`bg-primary-red`, text: "MAIN STAGE", caps: "MAIN" };
      case "sub":
        return { style: tw`bg-primary-blue`, text: "SUB STAGE", caps: "SUB" };
      case "live":
        return { style: tw`bg-primary-purple`, text: "LIVE STAGE", caps: "LIVE" };
      case "game":
        return { style: tw`bg-primary-green`, text: "GAME EVENT", caps: "GAME" };
      // no default
    }
  }, [stage]);
  return (
    <TimeTableHeadBox css={memo.style}>
      <h2 css={tw`font-zen font-bold text-white text-3xl hidden lg:inline select-none`}>{memo.text}</h2>
      <h2 css={tw`font-zen font-bold text-white text-base lg:hidden select-none`}>{memo.caps}</h2>
    </TimeTableHeadBox>
  );
};

const TimeTableColumn: FC<TimeTableColumnProperties> = ({ events, intervals, stage }) => (
  <section css={tw`space-y-2.25 md:space-y-6`} aria-label={stage}>
    <TimeTableHead stage={stage} />
    <TimeTableRow events={events} intervals={intervals} stage={stage} />
  </section>
);

const TimeTableAxis: FC<TimeTableAxisProperties> = ({ axes }) => (
  <section css={tw`grid grid-rows-7 h-time-table-row gap-0.5 md:(h-time-table-row-md gap-3) self-end`}>
    {axes.map(axis => (
      <time css={tw`font-zen font-bold text-gray-600 text-xss md:text-base lg:text-3xl select-none`} key={axis}>
        {axis}
      </time>
    ))}
  </section>
);

const TimeTable: FC<TimeTableProperties> = ({ events }) => (
  <section aria-label="Time table" css={tw`grid grid-cols-timetable gap-1 lg:gap-10`}>
    <TimeTableAxis axes={["09:00", "10:00", "11:00", "12:00", "13:00", "14:00", "15:00", "16:00"]} />
    <TimeTableColumn events={events.main} intervals={events.interval.main} stage="main" />
    <TimeTableColumn events={events.sub} intervals={events.interval.sub} stage="sub" />
    <TimeTableColumn events={events.live} intervals={events.interval.live} stage="live" />
    <TimeTableColumn events={events.game} intervals={events.interval.game} stage="game" />
  </section>
);

export { TimeTable };
