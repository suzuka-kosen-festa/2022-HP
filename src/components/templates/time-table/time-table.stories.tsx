import type { ComponentStoryObj, ComponentMeta } from "@storybook/react";
import { ComponentPropsWithoutRef } from 'react';
import { TimeTable } from ".";

type T = typeof TimeTable;
type Story = ComponentStoryObj<T>;
type Meta = ComponentMeta<T>;

const args: ComponentPropsWithoutRef<T> = {
  events: {
    main: [
      {
        id: 1,
        title: "オープニング",
        descriptions: "空白の2年間、たくさんの協力を得て遂に開催！溜め込んだものを全て出し切れ！高専祭開幕です！",
        date: "2022-10-30 09:00",
        venue: "メインステージ",
        start_time: "2022-10-30 09:00",
        end_time: "2022-10-30 09:30",
        stage: "main",
      },
      {
        id: 2,
        title: "音楽部コンサート",
        descriptions: "音楽部による、メインステージ最初を飾るセレモニー！盛り上がっていきましょー！",
        date: "2022-10-30 09:30",
        venue: "メインステージ",
        start_time: "2022-10-30 09:30",
        end_time: "2022-10-30 10:00",
        stage: "main",
      },
      {
        id: 3,
        title: "常識力バトル",
        descriptions: "高専生の常識をクイズ形式で競うバトル！高専生としての常識がある学生は誰か！？",
        date: "2022-10-30 10:00",
        venue: "メインステージ",
        start_time: "2022-10-30 10:00",
        end_time: "2022-10-30 10:30",
        stage: "main",
      },
      {
        id: 4,
        title: "勝山組",
        descriptions: "今年もよさこい勝山組の参加！鈴フェス大賞のよさこいを見逃すな！",
        date: "2022-10-30 10:45",
        venue: "メインステージ",
        start_time: "2022-10-30 10:45",
        end_time: "2022-10-30 11:15",
        stage: "main",
      },
      {
        id: 5,
        title: "へぇ王",
        descriptions: "先生達が自分の研究を紹介！4年生は卒検部を考えるいい機会に！高専生、中学生に向けた勉強イベント！",
        date: "2022-10-30 11:30",
        venue: "メインステージ",
        start_time: "2022-10-30 11:30",
        end_time: "2022-10-30 12:00",
        stage: "main",
      },
      {
        id: 6,
        title: "腕1GP",
        descriptions: "高専祭で腕相撲の1番を決める！一般参加ありの白熱バトルイベント！",
        date: "2022-10-30 12:30",
        venue: "メインステージ",
        start_time: "2022-10-30 12:30",
        end_time: "2022-10-30 13:00",
        stage: "main",
      },
      {
        id: 7,
        title: "TTTGP",
        descriptions:
          "その名もTop of The TikTok GP！TikTokで流行ったダンス、音源、Tik Tokerなど TikTokに関するクイズを出題！",
        date: "2022-10-30 13:15",
        venue: "メインステージ",
        start_time: "2022-10-30 13:15",
        end_time: "2022-10-30 13:45",
        stage: "main",
      },
      {
        id: 8,
        title: "EXBALLON",
        descriptions: "仲間を信じて、風船割って、四字熟語をより早く当てるチーム競技！！",
        date: "2022-10-30 14:00",
        venue: "メインステージ",
        start_time: "2022-10-30 14:00",
        end_time: "2022-10-30 14:45",
        stage: "main",
      },
    ],
    sub: [
      {
        id: 1,
        title: "アコギ部ライブ",
        descriptions:
          "コロナ禍でライブが出来ない中でも日々の練習を経て実力を高めてきました！その努力を是非聞いてください！",
        date: "2022-10-30 09:30",
        venue: "サブステージ",
        start_time: "2022-10-30 09:30",
        end_time: "2022-10-30 11:00",
        stage: "sub",
      },
      {
        id: 2,
        title: "出張Radio",
        descriptions: "ラジオパーソナリティの御三方がサブステージに出張！たくさんのコーナーを開催します！乞うご期待！",
        date: "2022-10-30 11:00",
        venue: "サブステージ",
        start_time: "2022-10-30 11:00",
        end_time: "2022-10-30 11:30",
        stage: "sub",
      },
      {
        id: 3,
        title: "乃木坂イントロクイズ",
        descriptions: "乃木坂限定のイントロクイズ！2年連続優勝を果たしたチャンピオンも出場！今年はいかに！？",
        date: "2022-10-30 11:30",
        venue: "サブステージ",
        start_time: "2022-10-30 11:30",
        end_time: "2022-10-30 12:00",
        stage: "sub",
      },
      {
        id: 4,
        title: "アコギ部ライブ",
        descriptions:
          "コロナ禍でライブが出来ない中でも日々の練習を経て実力を高めてきました！その努力を是非聞いてください！",
        date: "2022-10-30 12:30",
        venue: "サブステージ",
        start_time: "2022-10-30 12:30",
        end_time: "2022-10-30 14:30",
        stage: "sub",
      },
      {
        id: 5,
        title: "出張Radio",
        descriptions: "ラジオパーソナリティの御三方がサブステージに出張！たくさんのコーナーを開催します！乞うご期待！",
        date: "2022-10-30 15:00",
        venue: "サブステージ",
        start_time: "2022-10-30 15:00",
        end_time: "2022-10-30 15:45",
        stage: "sub",
      },
    ],
    live: [
      {
        id: 1,
        title: "Lana meer",
        descriptions:
          "練習以上の演奏ができるように頑張ります!みなさん観に来てください!アピールポイントは心に響く演奏ですー!",
        date: "2022-10-30 11:30",
        venue: "ライブステージ",
        start_time: "2022-10-30 11:30",
        end_time: "2022-10-30 12:00",
        stage: "live",
      },
      {
        id: 2,
        title: "59.9",
        descriptions: "何かとは言いませんが 60 の壁を越えられなかった人たちです。ギリギリで生きている方は是非。",
        date: "2022-10-30 12:15",
        venue: "ライブステージ",
        start_time: "2022-10-30 12:15",
        end_time: "2022-10-30 12:45",
        stage: "live",
      },
      {
        id: 3,
        title: "さうし～きゃっと",
        descriptions:
          "吉武:音楽さいこーー高専祭さいこーー\nどこよりも湧かせます\n後藤:僕らが奏でる音をたのしんで\n来田:ダブルシャウエッセン 1000",
        date: "2022-10-30 13:00",
        venue: "ライブステージ",
        start_time: "2022-10-30 13:00",
        end_time: "2022-10-30 13:30",
        stage: "live",
      },
      {
        id: 4,
        title: "あどばん",
        descriptions: "ありぺー!!!うけだ!!!こーせー!!!まえけん!!!\nみんな揃ってギニュー特戦隊!!!",
        date: "2022-10-30 13:45",
        venue: "ライブステージ",
        start_time: "2022-10-30 13:45",
        end_time: "2022-10-30 14:15",
        stage: "live",
      },
    ],
    game: [
      {
        id: 1,
        title: "大乱闘スマッシュブラザーズSPECIAL",
        descriptions: "ガノンドルフ最強",
        date: "2022-10-30 09:30",
        venue: "ゲームイベント",
        start_time: "2022-10-30 09:30",
        end_time: "2022-10-30 13:45",
        stage: "game",
      },
    ],
    interval: {
      main: [2, 2, 2, 1, 2, 1, 2, 2, 2, 1, 2, 1, 3, 5],
      sub: [2, 6, 2, 2, 2, 8, 2, 3, 1],
      live: [10, 2, 1, 2, 1, 2, 1, 2, 7],
      game: [2, 17, 9],
    },
  },
};

export default {
  args,
  argTypes: {
    events: {
      description: "Data of event",
      control: {
        type: "array",
      },
    },
  },
  component: TimeTable,
} as Meta;

export const Default: Story = {};
