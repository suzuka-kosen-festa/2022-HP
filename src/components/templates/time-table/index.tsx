import type { FC } from "react";
import tw from "twin.macro";
import type { TimeTableProperties } from "./types/model";
import {Title} from "../../molecules/title";
import {Switching} from "../../atoms/switching";
import {TimeTable as TT} from "../../organisms/time-table";

const Main = tw.main`pt-1 pb-8 sm:(pt-6 pb-40)`;

const TimetablePageBox = tw.div`flex flex-col justify-center items-center space-y-6 xl:space-y-32 3xl:space-y-48`;

const buttonData = [
  {
    buttonText: "10.30",
    query: "30",
  },
  {
    buttonText: "10.31",
    query: "31",
  },
];

const TimeTable: FC<TimeTableProperties> = ({ events }) => (
  <Main aria-label="Time table">
    <Title colorKey="yellow">Time schedule</Title>
    <TimetablePageBox>
      <Switching buttonData={buttonData} color="yellow" />
      <TT events={events} />
    </TimetablePageBox>
  </Main>
);

export { TimeTable };
