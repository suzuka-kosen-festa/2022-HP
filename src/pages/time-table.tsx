import type { FC } from "react";
import useSWR from "swr";
import { Redirect } from "wouter";
import { TimeTable } from "../components/templates/time-table";
import { fetcher } from "../libs/axios";
import type {SeparationEventList } from "../api/liveevent"

const TimeTablePage: FC = () => {
  const query = new URLSearchParams(window.location.search);
  const isThirty = query.get("select") === "30";
  const { data, error } = useSWR<SeparationEventList>(isThirty ? "/liveevent/2022-10-30" : "/liveevent/2022-10-31", fetcher)
  if (error) return <Redirect to="/error" />;
  if (!data) return <p>Loading...</p>;
  return <TimeTable events={data} />
}

export { TimeTablePage }
