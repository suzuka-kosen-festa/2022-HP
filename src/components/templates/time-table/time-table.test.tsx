import { composeStories } from "@storybook/testing-react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import * as stories from "./time-table.stories";

const { Default } = composeStories(stories);

const options = {
  name: "Time table",
};

describe("(components) templates/time-table", () => {
  beforeAll(() => {
    const portalRoot = document.createElement("div");
    portalRoot.setAttribute("id", "modal");
    document.body.appendChild(portalRoot);
  });
  test("to be templates", () => {
    const { container } = render(<Default />);
    expect(container).toBeTemplate();
  });
  test("to be [role=main]", () => {
    const { getByRole } = render(<Default />);
    expect(getByRole("main", options)).toBeInTheDocument();
  });
  test("take snap shot", () => {
    const { container } = render(<Default />);
    expect(container).toMatchSnapshot();
  });
});
