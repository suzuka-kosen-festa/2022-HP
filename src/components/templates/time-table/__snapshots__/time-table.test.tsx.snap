// Vitest Snapshot v1

exports[`(components) templates/time-table > take snap shot 1`] = `
<div>
  <main
    aria-label="Time table"
    class="css-10jk6q1"
  >
    <div
      class="css-79elbk"
    >
      <img
        alt="ink"
        class="css-mhf7w5-Title"
        height="580"
        loading="eager"
        src="/statics/Ink3_yellow.webp"
        width="592"
      />
      <h1
        class="css-1ueto1"
      >
        Time schedule
      </h1>
    </div>
    <div
      class="css-1x9od01"
    >
      <div
        class="css-1qq4q19"
      >
        <button
          class="css-1vjtn3a-SwitchingButton"
          tabindex="0"
        >
          <p
            class="css-1as8kh0-SwitchingButton"
          >
            10.30
          </p>
        </button>
        <button
          class="css-1vjtn3a-SwitchingButton"
          tabindex="0"
        >
          <p
            class="css-1as8kh0-SwitchingButton"
          >
            10.31
          </p>
        </button>
      </div>
      <section
        aria-label="Time table"
        class="css-1uyb8rf-TimeTable"
      >
        <section
          class="css-paib3t-TimeTableAxis"
        >
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            09:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            10:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            11:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            12:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            13:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            14:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            15:00
          </time>
          <time
            class="css-1wxgwcb-TimeTableAxis"
          >
            16:00
          </time>
        </section>
        <section
          aria-label="main"
          class="css-ebjbbp-TimeTableColumn"
        >
          <div
            class="css-1sgny0f-TimeTableHead"
          >
            <h2
              class="css-1fj6a01-TimeTableHead"
            >
              MAIN STAGE
            </h2>
            <h2
              class="css-1cgbium-TimeTableHead"
            >
              MAIN
            </h2>
          </div>
          <ul
            class="css-xfil3u-TimeTableRow"
          >
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  09:00
                </span>
                09:30
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                オープニング
              </p>
            </li>
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  09:30
                </span>
                10:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                音楽部コンサート
              </p>
            </li>
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  10:00
                </span>
                10:30
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                常識力バトル
              </p>
            </li>
            <div />
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  10:45
                </span>
                11:15
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                勝山組
              </p>
            </li>
            <div />
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  11:30
                </span>
                12:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                へぇ王
              </p>
            </li>
            <div />
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  12:30
                </span>
                13:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                腕1GP
              </p>
            </li>
            <div />
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  13:15
                </span>
                13:45
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                TTTGP
              </p>
            </li>
            <div />
            <li
              class="css-1melwrt-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  14:00
                </span>
                14:45
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                EXBALLON
              </p>
            </li>
            <div />
          </ul>
        </section>
        <section
          aria-label="sub"
          class="css-ebjbbp-TimeTableColumn"
        >
          <div
            class="css-4x186p-TimeTableHead"
          >
            <h2
              class="css-1fj6a01-TimeTableHead"
            >
              SUB STAGE
            </h2>
            <h2
              class="css-1cgbium-TimeTableHead"
            >
              SUB
            </h2>
          </div>
          <ul
            class="css-1dzet44-TimeTableRow"
          >
            <div />
            <li
              class="css-rep3tn-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  09:30
                </span>
                11:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                アコギ部ライブ
              </p>
            </li>
            <li
              class="css-rep3tn-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  11:00
                </span>
                11:30
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                出張Radio
              </p>
            </li>
            <li
              class="css-rep3tn-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  11:30
                </span>
                12:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                乃木坂イントロクイズ
              </p>
            </li>
            <div />
            <li
              class="css-rep3tn-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  12:30
                </span>
                14:30
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                アコギ部ライブ
              </p>
            </li>
            <div />
            <li
              class="css-rep3tn-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  15:00
                </span>
                15:45
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                出張Radio
              </p>
            </li>
            <div />
          </ul>
        </section>
        <section
          aria-label="live"
          class="css-ebjbbp-TimeTableColumn"
        >
          <div
            class="css-1ixslyp-TimeTableHead"
          >
            <h2
              class="css-1fj6a01-TimeTableHead"
            >
              LIVE STAGE
            </h2>
            <h2
              class="css-1cgbium-TimeTableHead"
            >
              LIVE
            </h2>
          </div>
          <ul
            class="css-1904nmv-TimeTableRow"
          >
            <div />
            <li
              class="css-1xh0q70-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  11:30
                </span>
                12:00
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                Lana meer
              </p>
            </li>
            <div />
            <li
              class="css-1xh0q70-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  12:15
                </span>
                12:45
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                59.9
              </p>
            </li>
            <div />
            <li
              class="css-1xh0q70-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  13:00
                </span>
                13:30
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                さうし～きゃっと
              </p>
            </li>
            <div />
            <li
              class="css-1xh0q70-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  13:45
                </span>
                14:15
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                あどばん
              </p>
            </li>
            <div />
          </ul>
        </section>
        <section
          aria-label="game"
          class="css-ebjbbp-TimeTableColumn"
        >
          <div
            class="css-1j8luaq-TimeTableHead"
          >
            <h2
              class="css-1fj6a01-TimeTableHead"
            >
              GAME EVENT
            </h2>
            <h2
              class="css-1cgbium-TimeTableHead"
            >
              GAME
            </h2>
          </div>
          <ul
            class="css-3iecga-TimeTableRow"
          >
            <div />
            <li
              class="css-11jeojo-TimeTableElement"
            >
              <time
                class="css-cj0c87-TimeTableElement"
              >
                <span
                  class="css-zu2x60-TimeTableElement"
                >
                  09:30
                </span>
                13:45
              </time>
              <p
                class="css-1e4vuyv-TimeTableElement"
              >
                大乱闘スマッシュブラザーズSPECIAL
              </p>
            </li>
          </ul>
        </section>
      </section>
    </div>
  </main>
</div>
`;
